src/components/ListCard/ListCard.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { BOX_BORDER_RADIUS } from '../../globalStyles/sharedStyles/box';
+import styled from 'styled-components';
+import CheckmarkIcon from '../../images/svg/checkmark.svg';
+
+type ListCardProps = {
+  title: string;
+  points: string[];
+};
+
+const CardComponent = styled.div`
+  width: 100%;
+  padding: 36px 48px;
```

Comment:
not critical, but maybe change padding on mobile

![image](https://github.com/vazco/forminer/assets/49162755/6e34aa3b-2542-4479-b1b9-971939a58cae)